### PR TITLE
Unminimize Event Details Dialog on update

### LINKED ISSUE #2019: When the Event Details Dialog is minimized, it is updated in the Background
Let's make sure issue is not already fixed in latest builds first.

- [x] I verified [latest Integration Build of Eclipse SDK](https://download.eclipse.org/eclipse/downloads/) doesn't provide the feature.

## Suggestion

When you double-click an entry in the Error Log, you get the well known Event Details Dialog:
grafik

When you double-click a new entry in the Error Log, the already opened Dialog is updated.

But when the Dialog is minimized, it is still updated but not visible to the user.
So the User clicks on an entry and from his perspective nothing happens. 

I would like to add an automatic "deminimize" so the dialog is again visible to the user.

## Community

- [x] I understand suggesting an enhancement doesn't mandate anyone to implement it. Other contributors may consider this suggestion, or not, at their own convenience. The most efficient way to get it fixed is that I implement it myself and contribute it back as a good quality patch to the project.

### PR DESCRIPTION
**Pull request by N1k145**
Fixes https://github.com/eclipse-platform/eclipse.platform.ui/issues/2019

When the Event Details Dialog is minimized, it is brought back up when the user Double-Clicks a new entry in the Error Log

### CODE REVIEW

**Review by BeckerWdf — APPROVED** [review]

**Comment by N1k145** [comment]
@BeckerWdf thank you for the review.
I don't have permission to merge myself.
Do you want to wait for a second review? If not, you can merge it from my side.

**Comment by vogella** [comment]
What about the Mac failure?

**Comment by N1k145** [comment]
@vogella looks like this was unrelated as the build now passes

**Comment by vogella** [comment]
Thanks @N1k145, great to see you continuing to contribute. 

Also I see you are a Gloomhaven fan (https://github.com/N1k145/gloomhavensecretariat), great game, shame that the app is based on Electon and not Flutter otherwise I would be very tempted to help with it (and finally finish Jaws of the Lion).

**Comment by N1k145** [comment]
@vogella thank you for reviewing and merging.

I have to say that is only my fork, as I contributed some data there. I have no influence on the language or framework ;)
But yes, it is a great game. And sometime I have to continue with Frosthaven :D

**Pull request merged**
